examples/extra_features_from_featurizer.ipynb

```diff
@@ -72,7 +187,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
```

Comment:
I would add a note to this cell that this is just an example because `MorganBinaryFeaturzer` in `featurizers/molecule.py` already implements this `MorganFingerprintMoleculeFeaturizer`.